build/template-publish-packages-and-symbols.yaml

```diff
@@ -34,10 +34,12 @@ steps:
     publishVstsFeed: '46419298-b96c-437f-bd4c-12c8df7f868d'
     allowPackageConflicts: true
   continueOnError: true
+  condition: eq(variables['PublishToVSTSFeed'], 'true')
 
 - task: PublishSymbols@2
   displayName: 'Publish symbols'
   inputs:
     SearchPattern: '${{ parameters.SymbolPublishWildcard }}'
     IndexSources: false
     SymbolServerType: TeamServices
+  condition: eq(variables['PublishSymbols'], 'true')
```

Comment:
do we also have a PublishSymbols variable in the build definition?

Reply:
yes